src/msg/async/net_handler.cc

```diff
@@ -132,6 +132,26 @@ int NetHandler::generic_connect(const entity_addr_t& addr, bool nonblock)
   return s;
 }
 
+/* Try to reconnect the socket.
+ *
+ * @return    0         success
+ *            > 0       just break, and wait for event
+ *            < 0       need to goto fail
+ */
+int NetHandler::reconnect(const entity_addr_t &addr, int sd)
```

Comment:
doxygen comments are normally put into the header file. and should be started with `/**`, see src/mon/Monitor.h for more examples.